src/components/app.js

```diff
@@ -220,6 +225,9 @@ const App = () => {
                         onLogoClick={() => onLogoClicked()}
                         user={user}
                         appsAndUrls={appsAndUrls}
+                        onThemeClick={handleThemeClick}
+                        theme={theme}
+                        onAboutClick={() => console.debug('about')}
```

Comment:
maybe we should open an about popup instead of console.debug ?

Reply:
"About" button is disable on commons ui side as it is not implement yet. We don't know yet what we want to put in the About section. It will be done later I guess

Reply:
I can remove the console.debug if we don't want it